source/distributor/service_impl.cc

```diff
@@ -0,0 +1,117 @@
+#include "distributor/service_impl.h"
+
+#include <grpc++/grpc++.h>
+
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "external/envoy/source/common/common/assert.h"
+#include "external/envoy/source/common/protobuf/message_validator_impl.h"
+#include "external/envoy/source/common/protobuf/utility.h"
+
+#include "api/distributor/distributor.pb.validate.h"
+
+namespace Nighthawk {
+
+::grpc::Status NighthawkDistributorServiceImpl::validateRequest(
```

Comment:
can this be a private function in an anonymous namespace, rather than a private method object?

Reply:
done in 6471b8687332bcae77a637c8b1d86640299e20c6